src/index.js

```diff
@@ -30,39 +30,44 @@ class ESLintWebpackPlugin {
       files: parseFiles(this.options.files, this.getContext(compiler)),
       extensions: arrify(this.options.extensions),
     };
+    Object.freeze(options);
```

Comment:
Why?

Reply:
ah, this was leftover from an exploration. I'll remove it.